### ISSUE TITLE
HtmlSettings.RemoveQuotedAttributes has a diabolical name

### ISSUE THREAD

**Issue by PrzemyslawKlys** [trullock/NUglify#191]
I'm opening this up, but not sure if you want to pursue this, as I believe you mentioned earlier where formatting JavaScript removes some unnecessary chars. 

Formatted

```html
        <meta content="width=device-width, initial-scale=1" name=viewport />
        <meta name=author />
        <meta content="2019-08-11 21:12:31" name=revised />
```

Before:

```html
        <meta content="width=device-width, initial-scale=1" name="viewport" />
        <meta name="author" />
        <meta content="2019-08-11 21:12:31" name="revised" />
```

**Comment by trullock** [comment]
You want the extremely badly named `HtmlSettings.RemoveQuotedAttributes`

This should probably be `false` for `HtmlSettings.Pretty()` by default, so thats what I'll do in this tickets as well as renaming the property

**Comment by trullock** [comment]
Fixed in 1.10.0
